src/server/services/procedures/this-x-does-not-exist/this-x-does-not-exist.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Provides access to images from various websites typically called this<X>doesnotexist.com.
+ * These are images of X generated from random noise by (typically) a GAN.
+ * 
+ * @alpha
+ * @service
+ * @category Science
```

Comment:
Maybe it would be good to create a new category called "ArtificialIntelligence". It will be a little sparse right now but we could add ParallelDots and Translation there, too. (Services can belong to more than one category, of course.)

Reply:
I would probably only edit this service for this PR and then update the others in another PR.